### PR TITLE
The one that gets the logos 'on point'

### PR DESCRIPTION
**Pull request by sturobson**
- Removes `--extreme` responsive version.
- Makes the maximum height 3rem, rather than 6rem.
- Removes some unneeded CSS.

### CODE REVIEW

File: components/embl-logo/embl-logo.scss [hunk 1]
```diff
@@ -12,20 +12,13 @@
 .embl-logo {
   background-image: url('../assets/embl-logo/assets/logo-medium.svg');
```
Comment (khawkins98) [inline components/embl-logo/embl-logo.scss:13]: Design mentioned that this should be just plan `/logo.svg` except for on tablet/mobile, is that worth addressing in this PR?

Comment (khawkins98) [reply components/embl-logo/embl-logo.scss:13]: Oh, wait, it looks like that should now work. Disregard!

**Review by khawkins98 — APPROVED** [review]

File: components/embl-logo/embl-logo.scss (hunk 1 above)
Comment (sturobson) [reply components/embl-logo/embl-logo.scss:13]: Oh, so the "more dots" one for everything? I thought it was less dots on mobile/tablet -- which this is.

Comment (khawkins98) [reply components/embl-logo/embl-logo.scss:13]: It's "more dots on desktop". Quickly scanning your code, it looks like you have it right.

Comment (sturobson) [reply components/embl-logo/embl-logo.scss:13]: yeah - I think the SVGs aren't named well. Perhaps that's a `N.x.x` breaking change to add to the todo list

**Pull request merged**